subsys/nrf_security/src/drivers/nrf_oberon/Kconfig

```diff
@@ -1,353 +1,1114 @@
-	  Please see https://www.keylength.com/
+	depends on PSA_WANT_ALG_SRP_6
+	depends on (PSA_WANT_ALG_SHA_1   && !PSA_ACCEL_SRP_6_3072_SHA1)   || \
+			   (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_SRP_6_3072_SHA224) || \
+			   (PSA_WANT_ALG_SHA_256 && !PSA_ACCEL_SRP_6_3072_SHA256) || \
+			   (PSA_WANT_ALG_SHA_384 && !PSA_ACCEL_SRP_6_3072_SHA384) || \
+			   (PSA_WANT_ALG_SHA_512 && !PSA_ACCEL_SRP_6_3072_SHA512)
 
-config PSA_CRYPTO_DRIVER_ALG_JPAKE_OBERON
+config PSA_NEED_OBERON_SRP_DRIVER
 	bool
-	prompt "PSA J-PAKE support - oberon" if !PSA_PROMPTLESS
-	default y if !PSA_DEFAULT_OFF && PSA_WANT_ALG_JPAKE
-	select EXPERIMENTAL
-	depends on PSA_CORE_OBERON
+	default y
+	depends on PSA_NEED_OBERON_SRP_6_3072
+
+# Oberon RSA Driver
 
-config PSA_CRYPTO_DRIVER_ALG_SPAKE2P_OBERON
+config PSA_NEED_OBERON_RSA_KEY_SIZE_1024
 	bool
-	prompt "PSA SPAKE2+ support - oberon" if !PSA_PROMPTLESS
-	default y if !PSA_DEFAULT_OFF && PSA_WANT_ALG_SPAKE2P
-	select EXPERIMENTAL
-	depends on PSA_CORE_OBERON
+	default y
+	depends on PSA_WANT_RSA_KEY_SIZE_1024
+	depends on (PSA_WANT_ALG_RSA_PKCS1V15_CRYPT && !PSA_ACCEL_RSA1024_PKCS1V15_CRYPT) \
+			|| (PSA_WANT_ALG_RSA_OAEP && ((PSA_WANT_ALG_SHA_1   && !PSA_ACCEL_RSA1024_SHA1_OAEP)     || \
+										  (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA1024_SHA224_OAEP)  || \
+										  (PSA_WANT_ALG_SHA_256 && !PSA_ACCEL_RSA1024_SHA256_OAEP)  || \
+										  (PSA_WANT_ALG_SHA_384 && !PSA_ACCEL_RSA1024_SHA384_OAEP)  || \
+										  (PSA_WANT_ALG_SHA_512 && !PSA_ACCEL_RSA1024_SHA512_OAEP))) \
+			|| (PSA_WANT_ALG_RSA_PSS && ((PSA_WANT_ALG_SHA_1   && !PSA_ACCEL_RSA1024_SHA1_PSS)   || \
+										 (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA1024_SHA224_PSS) || \
+										 (PSA_WANT_ALG_SHA_256 && !PSA_ACCEL_RSA1024_SHA256_PSS) || \
+										 (PSA_WANT_ALG_SHA_384 && !PSA_ACCEL_RSA1024_SHA384_PSS) || \
+										 (PSA_WANT_ALG_SHA_512 && !PSA_ACCEL_RSA1024_SHA512_PSS)))  \
+			|| (PSA_WANT_ALG_RSA_PKCS1V15_SIGN && ((PSA_WANT_ALG_SHA_1  && !PSA_ACCEL_RSA1024_SHA1_PKCS1V15_SIGN)     || \
+												   (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA1024_SHA224_PKCS1V15_SIGN)  || \
+												   (PSA_WANT_ALG_SHA_256 && !PSA_ACCEL_RSA1024_SHA256_PKCS1V15_SIGN)  || \
+												   (PSA_WANT_ALG_SHA_384 && !PSA_ACCEL_RSA1024_SHA384_PKCS1V15_SIGN)  || \
+												   (PSA_WANT_ALG_SHA_512 && !PSA_ACCEL_RSA1024_SHA512_PKCS1V15_SIGN)))
 
-config PSA_CRYPTO_DRIVER_ALG_SRP_OBERON
+config PSA_NEED_OBERON_RSA_KEY_SIZE_1536
 	bool
-	prompt "PSA SRP support - oberon" if !PSA_PROMPTLESS
-	default y if !PSA_DEFAULT_OFF && PSA_WANT_ALG_SRP
-	select EXPERIMENTAL
-	depends on PSA_CORE_OBERON
+	default y
+	depends on PSA_WANT_RSA_KEY_SIZE_1536
+	depends on (PSA_WANT_ALG_RSA_PKCS1V15_CRYPT && !PSA_ACCEL_RSA1536_PKCS1V15_CRYPT) \
+			|| (PSA_WANT_ALG_RSA_OAEP && ((PSA_WANT_ALG_SHA_1   && !PSA_ACCEL_RSA1536_SHA1_OAEP)     || \
+										  (PSA_WANT_ALG_SHA_224 && !PSA_ACCEL_RSA1536_SHA224_OAEP)  || \
```

Comment:
Are you using a different tab-size than the coding standard?

This is way to far indented.


Also please be consisdent in where the || / && are placed.
We usually always put them before the line break.